cobigen/cobigen-core/src/test/java/com/devonfw/cobigen/unittest/config/ConfigurationFinderTest.java

```diff
@@ -0,0 +1,337 @@
+package com.devonfw.cobigen.unittest.config;
+
+import static com.github.stefanbirkner.systemlambda.SystemLambda.withEnvironmentVariable;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import com.devonfw.cobigen.api.constants.ConfigurationConstants;
+import com.devonfw.cobigen.impl.config.ConfigurationProperties;
+import com.devonfw.cobigen.impl.util.ConfigurationFinder;
+
+/**
+ * Class to test the method loadTemplateSetConfigurations in ConfigurationFinder
+ *
+ */
+public class ConfigurationFinderTest {
```

Comment:
The test class name is misleading. Needs to be changed to PropertiesReaderTest or similar.

Reply:
adjusted javadocs instead.